lib/mix/tasks/recode.ex

```diff
@@ -85,8 +85,7 @@ defmodule Mix.Tasks.Recode do
   defp opts!(opts) do
     case OptionParser.parse!(opts, @opts) do
       {opts, []} -> opts
-      {opts, [inputs]} -> Keyword.put(opts, :inputs, inputs)
-      {_opts, args} -> Mix.raise("#{inspect(args)} : Unknown")
```

Comment:
Note that there is a `OptionParser.parse!` call that already checks for unknown arguments before getting to this line.

Therefore, this `Mix.raise/1` was only called when there were multiple input files and can be removed. 😸